src/components/ui/Button/README.md

```diff
@@ -42,9 +42,11 @@ Use a primary button to draw attention to the most important action on the page.
 Minimal buttons are good when you don't want to draw too much attention to a button. Use them for secondary actions.
 
 ```jsx
-<Button primary>Sign up now</Button>
-<Button>Sign up sometime</Button>
-<Button minimal>Sign up eventually</Button>
+<div style={{display: 'flex', flexWrap: 'wrap'}}>
```

Comment:
I wonder if we should have a component for this in the styleguide? Or can we use emotion's `css` prop for it instead? I'll give it a go; hardly a big deal!

Reply:
This works! There was an error in your CSS that caused wrapping issues, but I've fixed it. Thanks!